src/VisualStudio/Core/Impl/CodeModel/FileCodeModel.cs

```diff
@@ -123,7 +123,7 @@ internal override void Shutdown()
                 // We don't want to block up file removal on the UI thread since we want that path to stay asynchronous.
                 CodeModelService.DetachFormatTrackingToBuffer(_invisibleEditor.TextBuffer);
 
-                State.ProjectCodeModelFactory.ScheduleDeferredCleanupTask(() => { _invisibleEditor.Dispose(); });
+                State.ProjectCodeModelFactory.ScheduleDeferredCleanupTask(cancellationToken => { _invisibleEditor.Dispose(); });
```

Comment:
@sharwell A comment to the effect that the cancellation token should _not_ be looked at here may be wise...

Reply:
➡️ https://github.com/dotnet/roslyn/pull/46484